src/resources/2025/PrismaticBeret.ts

```diff
@@ -317,9 +317,9 @@ export function buskAt(
   const taoMultiplier = have_($skill`Tao of the Terrapin`) ? 2 : 1;
   try {
     if (
-      taoMultiplier *
-        (getPower(equippedItem($slot`hat`)) +
-          getPower(equippedItem($slot`pants`))) +
+      getPower(equippedItem($slot`hat`)) * taoMultiplier +
+        getPower(equippedItem($slot`pants`)) *
+          (taoMultiplier + (have_($effect`Hammertime`) ? 3 : 0)) +
```

Comment:
Let's slice this logic out into a special function in case additional stuff is added; we see it repeated a lot of place here and it'll be more DRY.